certified-connectors/Viafirma/Readme.md

```diff
@@ -0,0 +1,13 @@
+## Viafirma Connector
```

Comment:
@anfuca Please follow the formatting of [the readme.md template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/readme.md) and provide the required information.